examples/video/outstream/pb-ve-outstream-radiant.html

```diff
@@ -0,0 +1,217 @@
+            </div>
+		</div>
+
+
+		<!--video code fencing-->
+
+		<!--header code example-->
+		<div class="pb-alert pb-alert-warning" style="width:60vw;">
+			<p style="color:#a94443"><b>Warning:</b>
+			Do not forget to exchange the placementId in the code examples with your own placementId!</p>
+		</div>
+
+		<div style="width:60vw;">
+			<h4>Place this code in the page header.</h4>
+<pre class="pb-code-hl" style="width:60vw;">
+<!--put javascript code here-->
+&lt;!--radiant code--&gt;
+&lt;script src="https://cdn.radiantmediatechs.com/rmp/5.5.6/js/rmp.debug.js"&gt;&lt;/script&gt;
+</pre>
+
+		</div>
+
+		<!--body code example-->
+		<div style="width:60vw;">
+			<h4>Place this code in the page body.</h4>
+<pre class="pb-code-hl" style="width:60vw;">
+<!--put body html and javascript here-->
+&lt;script&gt;
+	/* our app where we run player
+   our app variables */
+		var pbjs;
+		pbjs = pbjs || {};
+		pbjs.que = pbjs.que || [];
+		var debug = false;
+		if (typeof window.console === 'undefined' || typeof window.console.log === 'undefined' || typeof window.console.dir === 'undefined') {
+		  debug = false;
+		}
+		var videoId = 'video1';
+
+		/* Prebid video ad unit
+		   This is a working example but you must use your own settings/bidders for production
+		   More docs at https://prebid.org/prebid-video/video-overview.html */
+		var adUnits = [{
+		  code: videoId,
+		  mediaTypes: {
+		    video: {
+		      playerSize: [640, 480],
+		      context: 'outstream'
+		    }
+		  },
+		  bids: [{
+		    bidder: 'appnexus',
+		    params: {
+		      placementId: 13232385,
+		      video: {
+		        skippable: true,
+		        playback_method: ['auto_play_sound_off']
+		      }
+		    }
+		  }]
+		}];
+
+		pbjs.que.push(function () {
+		  pbjs.addAdUnits(adUnits);
+		  pbjs.requestBids({
+		    timeout: 1000,
+		    bidsBackHandler: function (bids) {
+		      if (debug) {
+		        window.console.dir(bids);
+		      }
+		      // we get the VAST XML from bids adResponse and pass it to our outstream player
+		      if (bids && bids[videoId] && bids[videoId].bids && bids[videoId].bids[0] && bids[videoId].bids[0].adResponse) {
+		        var adResponse = bids[videoId].bids[0].adResponse;
+		        if (adResponse.ad && adResponse.ad.video && typeof adResponse.ad.video.content === 'string') {
+		          var vastXml = bids[videoId].bids[0].adResponse.ad.video.content;
+		          if (vastXml !== '') {
+		            if (debug) {
+		              window.console.log(vastXml);
+		            }
+		            var settings = {
+		              licenseKey: 'Kl8lbz16cz1rN2l2enM9Z2M9NnZvMnllaT9yb201ZGFzaXMzMGRiMEElXyo=',
```

Comment:
Use license key Kl8lZ292K3N6MmVvZD9yb201ZGFzaXMzMGRiMEElXyo= for site prebid.org instead.